src/BloomExe/Book/Book.cs

```diff
@@ -1341,6 +1341,9 @@ private XmlElement FindPageDiv(IPage page)
 			//review: could move to page
 			var pageElement = OurHtmlDom.RawDom.SelectSingleNodeHonoringDefaultNS(page.XPathToDiv);
 			Require.That(pageElement != null,"Page could not be found: "+page.XPathToDiv);
+			if (pageElement != null)
+				pageElement.InnerXml = XmlHtmlConverter.RemoveEmptySelfClosingTags(pageElement.InnerXml);
+				
```

Comment:
It was empty self-closing tags that were causing the thumbnails to not line up correctly (doesn't really make sense to me, but...).